readme.md

```diff
@@ -1,2 +1,3 @@
 This is my read.md file
 I have made a change to this file
+I have also made a change to this file
```

Comment:
This is a load of rubbish Amanda I expected better from you

Reply:
not funny

Reply:
Im sorry maybe I was too harsh, lots of love